apps/rush-lib/src/logic/taskRunner/BaseBuilder.ts

```diff
@@ -24,21 +24,21 @@ export abstract class BaseBuilder {
   /**
    * Name of the task definition.
    */
-  abstract readonly name: string;
+  public abstract readonly name: string;
```

Comment:
Why did this have to change in this PR?

Reply:
This was caught by the rule `@typescript-eslint/explicit-member-accessibility`. Not sure why it wasn't caught before.... but it is now.